doc/Development_Documentation/18_Tools_and_Features/39_Sitemaps.md

```diff
@@ -11,12 +11,12 @@ an event handler and start to add entries.
 Sitemaps can either be exposed by being generated on-the-fly or by being dumped to static files. What to use depends on the size
 of your site (e.g. the size of the tree which needs to be processed). In general it's recommended to create static files
 as it reduces the overhead of creating the sitemap on every crawler request. If you want to serve the sitemap directly,
-you need to register the sitemaps routes in your routing config (see [PrestaSitemapBundle Documentation](https://github.com/prestaconcept/PrestaSitemapBundle/blob/master/Resources/doc/1-installation.md)
+you need to register the sitemaps routes in your routing config (see [PrestaSitemapBundle Documentation](https://github.com/prestaconcept/PrestaSitemapBundle/blob/3.x/doc/1-installation.md)
 for details).
 
 ```yaml
 PrestaSitemapBundle:
-    resource: "@PrestaSitemapBundle/Resources/config/routing.yaml"
+    resource: "@PrestaSitemapBundle/config/routing.yaml"
```

Comment:
Also mentioned in https://github.com/pimcore/pimcore/blob/10.x/doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md?